example/example/tests/test_views.py

```diff
@@ -39,3 +41,27 @@ def test_proxy_s3_view(api_client, mock_data_client):
     url = f"{reverse(view_name)}?{invalid_url_params}"
     response = api_client.get(url)
     assert response.status_code == status.HTTP_500_INTERNAL_SERVER_ERROR
+
+
+@pytest.mark.django_db
+class TestUserTracking:
+    def test_tracking_features(self, api_client):
+        log_data = [
+            {
+                "content": {
+                    "key": "Value 1",
+                },
+                "tags": ["dataset"]
+            }, {
+                "content": {
+                    "key": "Value 2",
+                },
+                "tags": ["dataset"]
+            }
+        ]
+
+        url = reverse("log-tracking")
+
+        response = api_client.post(url, log_data, format="json")
+
+        assert status.is_success(response.status_code)
```

Comment:
Another thing to test would be that there are now 2 log records each w/ a tag of "dataset" in the db

Reply:
Good idea, I will also add a further test for the error state you highlighted above, when it isn't a `list` being passed in.